tests/core/full_node/stores/test_full_node_store.py

```diff
@@ -265,7 +304,9 @@ async def test_basic_store(
                 result = await blockchain.get_sp_and_ip_sub_slots(block.header_hash)
                 assert result is not None
                 sp_sub_slot, ip_sub_slot = result
```

Comment:
missing 

  next_sub_slot_iters, next_difficulty = get_next_sub_slot_iters_and_difficulty(
                    blockchain.constants, False, sb, blockchain
                )
in line 306

Reply:
Thanks for looking this over.  I believe @AmineKhaldi noted this before as well and Arvid.  I'm glad you all are in agreement about it.  `:]`  We have now merged Amine's PR for this and I think this one can be closed.